requirements/constraints.txt

```diff
@@ -11,25 +11,13 @@
 # TODO: Many pinned dependencies should be unpinned and/or moved to this constraints file.
 
 # Use latest Django LTS version
-Django>=2.2,<2.3.0
-
-# django-model-utils version 4.0.0 dropped support to python 2.7	
-django-model-utils<4.0.0	
+Django>=2.2,<2.3.1
```

Comment:
Not that it should matter, but why the change in upper limit?

Reply:
Changed back to previous `Django>=2.2,<2.3.0`.